quota/etcd/quotapb/quotapb.proto

```diff
@@ -74,7 +74,9 @@ message Config {
 }
 
 // Sequencing-based replenishment strategy settings.
-// Only global and tree quotas may use sequencing-based replenishment.
+//
+// Only global/write and trees/write quotas may use sequencing-based
+// replenishment.
```

Comment:
Why doesn't this give a Travis error about the .pb.go not matching?

Reply:
We filtered `.pb.go` files out of that test: https://github.com/google/trillian/blob/master/.travis.yml#L73.

IIRC, it's because the generation isn't completely deterministic (sometimes fileDescriptors don't match).

I'll regen the protos before merging.